draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -561,7 +595,12 @@ in (#hash_function_claim)).
 
 ### Validating the Key Binding JWT
 
-To validate the signature on the Key Binding JWT, the Verifier MUST use the key material in the SD-JWT. If it is not clear from the SD-JWT, the Key Binding JWT MUST specify which key material the Verifier needs to use to validate the Key Binding JWT signature using JOSE header parameters such as `kid` and `x5c`.
+The Verifier MUST ensure that the key with which it validates the signature on
+the Key Binding JWT is the key specified in the SD-JWT as the Holder's public
+key.  For example, if the SD-JWT contains a `cnf` value of type `jwk`, the
+Verifier would confirm that the public key's JWK represents the same key as the
+`jwk` value.  Or if the SD-JWT contains a `cnf` value of type `jwk`, then the
+Verifier could parse the provided JWK and use it to verify the Key Binding JWT.
```

Comment:
these 4 lines now just aren't quite right (if jwk or jwk...) - here's an attempt to fix by shortening and simplifying: 

```suggestion
key.  For example, if the SD-JWT contains a `cnf` value with a `jwk` member, the
could parse the provided JWK and use it to verify the Key Binding JWT.
```